### ISSUE TITLE
Foot note with image

### ISSUE THREAD

**Issue by ddahora** [davidgohel/flextable#134]
Issues are a great way to keep track of tasks, enhancements, and bugs for your projects (https://guides.github.com/features/issues/). When submitting, you need to provide the 
following elements:

### Reporting a bug:

- [ ] The code that is producing the error, it has to be a minimal reproducible example.
Stackoverflow is providing good explanations about it: https://stackoverflow.com/help/mcve. You can use package `reprex` to help you: http://reprex.tidyverse.org/. The most popular R stackoverflow question is about the subject: https://stackoverflow.com/questions/5963269/how-to-make-a-great-r-reproducible-example.

- [ ] the results of R command `sessionInfo()`. It had to be executed after you loaded the packages used by your example. This will let me know what is your version of R and what are the versions of the packages you used in your example. 

- [ ] you did checked you had the latest version of the package on CRAN (and on github if issue exists with CRAN version).

- [ ] you searched in the *open* and *closed* issues on the github repository.

### Asking an enhancement or a new function

Explain the value, describe the situation.

Any suggestions to make the documentation better is really appreciated.

### Asking a question

If you are looking for help on how to use the package, please visit Stackoverflow and tag your 
question with `[r]` and `[flextable]`. I usually read them and answer when possible.

**Comment by davidgohel** [comment]
There is no question I can answer there. I will reopen if you can provide a description

**Comment by ddahora** [comment]
Hello,
created a table where the cells that had the maximum and minimum values of each row were highlighted respectively with the colors, red and green. I managed to make the table perfectly, but I have a problem to make the caption. With the help of the ionicons package, I made the following code that didn't work.

greensq<-as_png("stop",fill="palegreen",width = .3,height= .3)
redsq <-as_png("stop",fill="lightcoral",width = .3,height= .3)

table1 <- flextable::mk_par(table1,
value = as_paragraph(as_chunk(flextable::as_image(src = greensq),"Minimum rate",
"  ",flextable::as_image(src = redsq),"Maximum rate"),part="footer"))

Error in formater(x, ...) : could not find function "formater"
